docs/en/using-status/leave-a-group-chat.md

```diff
@@ -8,7 +8,31 @@ hide:
 
 # Leave a group chat
 
- :octicons-tools-24:
+If you don’t want to be a member of a [group chat][understand-group-chats] anymore, you can leave the chat. You can always [re-join the chat][join-a-group-chat] later if you change your mind.
 
-!!! note ""
-     We're working on this content.
+Once you leave, a note appears in the group chat's message history letting other members know that you left the chat. You can still see the previous message history for a group chat you left.
```

Comment:
> You can still see the previous message history for a group chat you left.

I'm sorry, I don't understand this. Do I still see the group chat, can I enter it and check my previous messages after leaving the chat?